test/Serilog.Tests/Settings/AppSettingsTests.cs

```diff
@@ -1,18 +1,18 @@
-﻿using System.Configuration;
-using NUnit.Framework;
+﻿#if !DNXCORE50
+using System.Configuration;
+using Xunit;
 using Serilog.Events;
 using Serilog.Tests.Support;
 
 namespace Serilog.Extras.AppSettings.Tests
 {
-    [TestFixture]
     public class AppSettingsTests
     {
-        [Test]
+        [Fact(Skip = "Fails on DNX451")]
```

Comment:
Is it possible to raise a ticket listing the skipped tests and what needs to happen to get them back to happy?